### PR TITLE
added assets insert

### PR DESCRIPTION
**Pull request by chandan5**

### CODE REVIEW

File: app/controllers/assets.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                $_POST['description']
+            );
+        }
+
+        $this->load_view('assets/add_assets', array(
+            'success' => $success
+        ));
+    }
+
+    function list_data() {
```
Comment (nisargjhaveri) [inline app/controllers/assets.php]: Change the name to `view` for consistency.

File: app/controllers/assets.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        $list = $this->assets_model->get_list();
+
+        $this->load_view('assets_list', array(
+            'list' => $list
+        ));
+    }
```
Comment (nisargjhaveri) [inline app/controllers/assets.php]: Please remove this function as it is not currently working. Also, the view `asset_list` is not there.

File: app/models/assets_model.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        $result = $this->DB->query("SELECT * FROM `contact_details`");
+        if ($result) {
+            $contact_list = $result->fetch_all(MYSQLI_ASSOC);
+            return $contact_list;
+        }
+        else {
+            return false;
+        }
+        */
+    }
```
Comment (nisargjhaveri) [inline app/models/assets_model.php]: Remove this also.

**Comment by nisargjhaveri** [comment]
:confused: A better commit message :/

**Comment by chandan5** [comment]
Next time I will take care.

**Comment by nisargjhaveri** [comment]
Please change this one also!?

**Comment by gauravmittal1995** [comment]
@chandan5 use `git commit --amend` for changing commit message.

**Comment by chandan5** [comment]
closing as last commits over this pr were wrong...

**Pull request closed without merging**